example/pubspec_overrides.yaml

```diff
@@ -0,0 +1,17 @@
+dependency_overrides:
```

Comment:
You excluded the dependency_overrides for the packages. Not sure if you want to exclude this for the example app too?

Reply:
Ah, good catch. Fixed!